rotkehlchen/data/importer.py

```diff
@@ -1275,3 +1323,99 @@ def import_uphold_transactions_csv(self, filepath: Path, **kwargs: Any) -> Tuple
                 except KeyError as e:
                     return False, str(e)
         return True, ''
+
+    def _consume_bisq_trade(self, csv_row: Dict[str, Any], **kwargs: Any) -> None:
+        """
+        Consume the file containing only trades from BlockFi. As per my investigations
+        (@yabirgb) this file can only contain confirmed trades.
+        - UnknownAsset
+        - DeserializationError
+        """
+        if csv_row['Status'] == 'Canceled':
```

Comment:
didn't you write in the docstring above that it shoudl contain only confirmed trades? It's totally fine, even good to check ofc, but wonder why you wrote the docstring comment now.

Reply:
Exactly what you thought the docstring was a copypasta